tests/spec-e2e/nightwatch-test.test.js

```diff
@@ -19,8 +19,8 @@ module.exports = {
       .mouseButtonDown(0)
       .getElementSize(selTimeline, function (result) {
         this.moveToElement(selTimeline, result.value.width, result.value.height / 2);
+        this..mouseButtonUp(0);
```

Comment:
The double dots error for me when testing the _test:e2e_ I guess this should be a single dot right?

Reply:
Yep

Reply:
Fixed